docs/simplesamlphp-upgrade-notes-2.0.md

```diff
@@ -15,18 +15,18 @@ Upgrade notes for SimpleSAMLphp 2.0
     to manually switch back the `sharedkey_algorithm`. Note that CBC is vulnerable to the Padding oracle attack.
 - In compliancy with SAML2INT, AuthnRequests that are signed will have their signature validated unless specifically disabled by setting `validate.authnrequest` to `false`.  If unset, or set to true, signatures will be validated and requests not passing validation will be refused.
 - The following classes have been migrated to non-static:
-  + lib/SimpleSAMLphp\Utils\Arrays
-  + lib/SimpleSAMLphp\Utils\Attributes
-  + lib/SimpleSAMLphp\Utils\Auth
-  + lib/SimpleSAMLphp\Utils\Config
-  + lib/SimpleSAMLphp\Utils\Crypto
-  + lib/SimpleSAMLphp\Utils\EMail
-  + lib/SimpleSAMLphp\Utils\HTTP
-  + lib/SimpleSAMLphp\Utils\Net
-  + lib/SimpleSAMLphp\Utils\Random
-  + lib/SimpleSAMLphp\Utils\System
-  + lib/SimpleSAMLphp\Utils\Time
-  + lib/SimpleSAMLphp\Utils\XML
+  + lib/SimpleSAML\Utils\Arrays
```

Comment:
What's with the `\` vs `/`?

Reply:
Ah, good catch!
Thanks for reviewing!  I'll try and raise coverage some more before merging